src/Cli/dotnet/commands/dotnet-workload/install/WorkloadInstallCommand.cs

```diff
@@ -176,7 +178,9 @@ public void InstallWorkloads(IEnumerable<WorkloadId> workloadIds, bool skipManif
                 workloadIds = workloadIds.Concat(installedWorkloads).Distinct();
 
                 _workloadManifestUpdater.UpdateAdvertisingManifestsAsync(includePreviews, offlineCache).Wait();
-                manifestsToUpdate = _workloadManifestUpdater.CalculateManifestUpdates().Select(m => (m.manifestId, m.existingVersion, m.newVersion));
+                manifestsToUpdate = string.IsNullOrWhiteSpace(_fromRollbackDefinition) ?
+                    _workloadManifestUpdater.CalculateManifestUpdates().Select(m => (m.manifestId, m.existingVersion, m.newVersion)) :
+                    _workloadManifestUpdater.CalculateManifestRollbacks(_fromRollbackDefinition);
```

Comment:
Related to @dsplaisted's point: https://github.com/dotnet/sdk/pull/21283#issuecomment-925369614

This looks like it either updates *or* uses the rollback file.

Would this potentially break if:
1. You installed some .NET SDK, let's say it's RC2 on release day.
2. You give it a rollback file such as:
```json
{"microsoft.net.sdk.android":"31.0.101-preview.9.6"}
```
```
> dotnet workload install android --from-rollback-file foo.json
```
3. Would the install pretty much fail every time, when you don't list `microsoft.net.workload.mono.toolchain`? It is a dependency of the `android` workload, and the version of the various Mono baseline manifests won't be on NuGet.org?